Sources/SQLKit/SQLDialect.swift

```diff
@@ -1,5 +1,5 @@
 public protocol SQLDialect {
-    var name: String { get }
+    var name: String { get set }
```

Comment:
This should be `{ get }`. I think this change was made to make `GenericDialect`'s `name` settable, but that shouldn't affect the protocol. If you know you have `GenericDialect` as opposed to a `SQLDialect`, Swift will let you mutate the property.